wagtail/contrib/postgres_search/backend.py

```diff
@@ -52,6 +52,9 @@ def prepare_field(self, obj, field):
             yield (field, get_weight(field.boost),
                    self.prepare_value(field.get_value(obj)))
 
+        elif isinstance(field, AutocompleteField):
+            yield (field, 'D', self.prepare_value(field.get_value(obj)))
```

Comment:
Is there a reason we're hard-coding boost "D" here, rather than calling `get_weight`? (If so, I'm fine with that, but we should probably explain why in a comment.)

Reply:
There's no boost on `AutocompleteField`. Autocomplete should be just one field (either title, name, etc)

Reply:
Is there a fundamental reason not to support multiple `AutocompleteField`s on a model? I could see a use for something like a Person model with AutocompleteFields on `first_name` and `last_name`, so that `Person.objects.autocomplete('matthe')` returns both Matthew Westcott and Bernard Matthews (and potentially gives the latter a higher ranking, if there's a boost on `last_name`).

(Happy to treat this as out of scope for the current PR though...)